Sources/Orbit/Components/ListChoice.swift

```diff
@@ -0,0 +1,252 @@
+import SwiftUI
+
+/// Shows one of a selectable list of items with similar structures.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/listchoice/)
+/// - Important: Component expands horizontally to infinity up to a ``Layout/readableMaxWidth``.
+public struct ListChoice: View {
+
+    let title: String
+    let description: String
+    let icon: Icon.Symbol
+    let trailingElement: ListChoice.TrailingElement
+    let backgroundColor: BackgroundColor?
+    let action: () -> Void
+
+    public var body: some View {
+        SwiftUI.Button(
+            action: {
+                HapticsProvider.sendHapticFeedback(.light(0.5))
+                action()
+            },
+            label: {
+                buttonContent
+            }
+        )
+        .buttonStyle(ListChoiceButtonStyle(backgroundColor: backgroundColor))
+        .accessibility(label: SwiftUI.Text(title))
+        .accessibility(hint: SwiftUI.Text(description))
+    }
+
+    @ViewBuilder var buttonContent: some View {
+        HStack(spacing: 0) {
+            header
+                .frame(maxWidth: .infinity, alignment: .leading)
+
+            // Provide minimal height (frame(minHeight:) collapses multiline text in snapshots)
+            Color.clear
+                .frame(width: 0, height: 48)
+
+            trailingElementView
+                .padding(.trailing, .medium)
+        }
+        .frame(maxWidth: Layout.readableMaxWidth, alignment: .leading)
+        .overlay(separator, alignment: .bottom)
+    }
+    
+    @ViewBuilder var header: some View {
+        if isHeaderEmpty == false {
+            HStack(alignment: .firstTextBaseline, spacing: .small) {
+                iconView
```

Comment:
Based on Figma, the spacing of icon is ~12 leading and ~8 trailing around the icon. To avoid custom dynamic paddings, it might be enough to wrap "icon + text content" into an additional HStack with a `.xSmall(8)` spacing and leave the outer padding to `.medium` as it is.